framework/include/eventloop/eventloop.h

```diff
@@ -68,59 +79,61 @@ enum el_event_type_e {
 };
 
 /**
- * @brief Return value of callback function for event
- * @details The callback function should return EVENTLOOP_CALLBACK_STOP(false) or EVENTLOOP_CALLBACK_CONTINUE(true).\n
- * It decides whether continuing to execute callback function for the event or not.\n
- * If it returns EVENTLOOP_CALLBACK_STOP, receiving registered events is stopped.\n
- * The registered handler won't be executed from now on although receiving the event.\n
- * Otherwise, a callback function is called continuously.
- */
-#define EVENTLOOP_CALLBACK_STOP 0
-#define EVENTLOOP_CALLBACK_CONTINUE 1
-
-/**
- * @brief EventLoop Timer structure
- */
-typedef uv_timer_t el_timer_t;
-
-/**
- * @brief EventLoop Loop structure
+ * @brief Handle Type of Event Loop
+ * @details User can set the mode of handle. \n
+ * If a task or pthread add handle with EL_MODE_WAIT_DONE, it doesn't exit until they added handles are processed.
  */
-typedef uv_loop_t el_loop_t;
-
-/**
- * @brief EventLoop Event structure
- */
-typedef uv_signal_t el_event_t;
+enum el_mode_e {
```

Comment:
1. updated
2. Yes, It is valid when task is running only. so it can be exited without execution if registered event doesn't occur when task is running.
3. It means that exit of task, and the logics about 'exit' here are called when task exits
 If there are some events with EL_MODE_WAIT_DONE when task exits, it processes them. If not, it cleans them including used resources.